tasks/flush.yml

```diff
@@ -4,11 +4,15 @@
   command: "php symfony tools:purge --no-confirmation --email='{{ atom_user_email }}' --username='{{ atom_user_username }}' --password='{{ atom_user_password }}' --title='{{ atom_title }}' --description='{{ atom_description }}' --url='http://undefined'"
   args:
     chdir: "{{ atom_path }}"
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/opt/rh/rh-php70/root/bin/"
 
```

Comment:
I guess adding this to the path doesn't affect when deploying on ubuntu, but is there a way to make this addition conditional (add to path only when deploying on CentOS/RH)?. Or just add a comment to clarify.